front-end/src/components/Admin/index.js

```diff
@@ -8,6 +8,7 @@ let index = 1;
 
 class Admin extends Component {
   state = {
+    recordData: '',
```

Comment:
Did you intend to save separate recordData for each tab? If yes, it should by within the 'tabs' array.